HW05_Julia.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "HW05_Julia"
+author: "Julia Shangguan"
+date: "8/1/2020"
+output: github_document 
+---
+
+```{r setup, include = TRUE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+Loading in necessary packages:
+```{r load libraries}
+#install.packages('tidyverse')
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+```
+
+## Part 1: Tidying the dad_mom file
+
+```{r tidy dad_mom data}
+parent_data <- read.delim("dad_mom.txt") #read dataframe
+
+parent_data_tidy <- parent_data %>%
+  gather(key = "parent", value= "name", name_dad, name_mom) %>% #create name column
+  gather(key= "key1", value = "income", income_dad, income_mom) %>% #create income column
+  select(-key1)
+
+parent_data_tidy$parent <- recode(parent_data_tidy$parent, "name_dad" = "dad", "name_mom" = "mom") #create parent's gender column
+
+parent_data_tidy
+```
+
+## Part 2: Joining together CTRP data
+
+```{r CTRP data}
+#read all dataframes
+cancer_cell_line <- read_csv("CTRP_files/cancer_cell_line_info.csv")
+AUC_1 <- read_csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read_csv("CTRP_files/AUC_2.csv")
+compound_info <- read_csv("CTRP_files/compound_info.csv")
+experiment_info <- read_csv("CTRP_files/Experiment_info.csv")
+```
+
+```{r}
+#looking at each dataframe
+glimpse(cancer_cell_line)
+glimpse(AUC_1)
+glimpse(AUC_2)
+glimpse(compound_info)
+glimpse(experiment_info)
+```
+```{r joining the tables}
+#is it wrong to use full_join to merge all the tables? or should I be using inner_join?
+
+AUC_all_compound<- AUC_1 %>%
+  full_join(AUC_2, by = c("experiment_id", "area_under_curve", "master_cpd_id")) %>%
+  full_join(compound_info, by = "master_cpd_id")
+
+ccl_experiment_AUC_compound <- cancer_cell_line %>%
+  full_join(experiment_info, by = "master_ccl_id") %>%
+  full_join(AUC_all_compound, by=c("expt_id"="experiment_id"))
```

Comment:
I copy/pasted your code into a new RMarkdown file, and your code had a few hundred more rows than mine. So there does seem to be a difference between the two, but I'm not sure which one was "better" in combining the data.